Package/Windows/SigningExclusions.json

```diff
@@ -14,6 +14,10 @@
   "Xamarin/Workbooks/Client/MouseKeyboardActivityMonitor.dll": "oss",
   "Xamarin/Workbooks/Client/Newtonsoft.Json.dll": "oss",
   "Xamarin/Workbooks/Client/YamlDotNet.dll": "oss",
+  "Xamarin/Workbooks/Client/SQLitePCLRaw.batteries_green.dll": "oss",
```

Comment:
Humanizer would need to be excluded as well (though I'd like to drop it if possible...)

Reply:
If it's not being detected, it must already be signed. I'll verify, though.

Reply:
Yeah, it's already signed.